ci/jenkins/test.sh

```diff
@@ -691,15 +697,24 @@ function run_e2e {
 
     mkdir -p "${WORKDIR}/.kube"
     mkdir -p "${WORKDIR}/.ssh"
-    cp -f "${WORKDIR}/kube.conf" "${WORKDIR}/.kube/config"
+    # cp -f "${WORKDIR}/kube.conf" "${WORKDIR}/.kube/config"
     generate_ssh_config
 
     set +e
     mkdir -p `pwd`/antrea-test-logs
     # HACK: see https://github.com/antrea-io/antrea/issues/2292
+    echo "debug 1"
     go mod edit -replace github.com/moby/spdystream=github.com/antoninbas/spdystream@v0.2.1 && go mod tidy
+    echo "debug 2"
```

Comment:
I guess you are still debugging the code, don't forget to remove unnecessary comments when you feel it's ready for review.